algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -0,0 +1,109 @@
+package com.hartwig.serve.sources.ckb;
+
+import java.util.List;
+import java.util.Set;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import com.hartwig.serve.ckb.datamodel.CkbEntry;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ClinicalTrial;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.Location;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.VariantRequirementDetail;
+import com.hartwig.serve.ckb.datamodel.indication.Indication;
+import com.hartwig.serve.datamodel.EvidenceDirection;
+import com.hartwig.serve.datamodel.EvidenceLevel;
+import com.hartwig.serve.datamodel.ImmutableTreatment;
+import com.hartwig.serve.datamodel.Knowledgebase;
+
+import org.jetbrains.annotations.NotNull;
+
+class ActionableTrialFactory implements ActionableEntryFactory {
+
+    private static final Set<String> POTENTIALLY_OPEN_RECRUITMENT_TYPES = Sets.newHashSet();
+    private static final Set<String> COUNTRIES_TO_INCLUDE = Sets.newHashSet();
+    private static final Set<String> VARIANT_REQUIREMENT_TYPES_TO_INCLUDE = Sets.newHashSet();
+
+    static {
+        POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("Recruiting");
+        POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("Active, not recruiting");
+        POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("Unknown status");
+        POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("RECRUITING");
+        POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("ACTIVE_NOT_RECRUITING");
+
+        COUNTRIES_TO_INCLUDE.add("Netherlands");
+        COUNTRIES_TO_INCLUDE.add("Belgium");
+        COUNTRIES_TO_INCLUDE.add("Germany");
+
+        VARIANT_REQUIREMENT_TYPES_TO_INCLUDE.add("partial - required");
+        VARIANT_REQUIREMENT_TYPES_TO_INCLUDE.add("required");
+        VARIANT_REQUIREMENT_TYPES_TO_INCLUDE.add("partial");
```

Comment:
I'd tend to think it would be better to make this case insensitive?

Reply:
Done!